src/test/java/com/google/gcp_variant_transforms/library/HeaderIteratorTest.java

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import com.google.gcp_variant_transforms.library.HeaderIterator;
+import static com.google.common.truth.Truth.*;
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+import org.junit.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Units tests for HeaderIterator.java
+ */
+public class HeaderIteratorTest {
+
+  @Test
+  public void testHeaderLinesHasNext_whenCheckingBoolean_thenTrue() {
+    // Arrange
```

Comment:
Comments //Arrange, //Act and //Assert do now have to be specified, just have a newline between each block.

Reply:
resolved